src/backend/clearboard/main.py

```diff
@@ -47,3 +96,124 @@ async def post_picture(file: UploadFile = File(...)):
+
+    except:
+        print('no file to send')
+
+
+@app.get("/original_photo")
+async def photo(roomName: Optional[str] = None):
+    try:
+        original_img_path = './' + roomName + '/' + roomName + '.jpg'
+        if os.path.exists(os.path.abspath(original_img_path)):
+            img = cv2.imread(original_img_path)
+            volume = np.asarray(img)
+            image = get_image(volume)
+            print('image sent')
+            return Response(content=image)
+        else:
+            print('file not found')
+            time.sleep(1)
+    except:
+        print('no file to send')
+
+
+@app.websocket("/ws1")
+async def websocket_endpoint(websocket: WebSocket):
+    await manager.connect(websocket)
+    temps = 0
+    temps_coord = 0
+    try:
+        while True:
+            print('test')
+            roomName = await websocket.receive_text()
+            original_img_path = './' + roomName + '/' + roomName + '.jpg'
+            img_cropped_path = './' + roomName + '/' + roomName + 'cropped.jpg'
+
+            if (os.path.isfile(os.path.abspath(original_img_path)) and (temps != os.path.getctime(original_img_path))):
+                temps = os.path.getctime(original_img_path)
+                temps_coord = os.path.getctime('./coord.npy')
+                print('change file')
+                try:
+                    coordinates = np.load("./coord.npy")
+                except:
+                    coordinates = None
+                parallax.crop(original_img_path, coordinates, img_cropped_path)
+                print('new photo to send')
+
+                await manager.send_personal_message("true", websocket)
+
+                if (os.path.isfile(os.path.abspath('./coord.npy'))) and (temps_coord != os.path.getctime('./coord.npy')):
+                    temps_coord = os.path.getctime('./coord.npy')
+                    print('change of coordinates')
+                    try:
+                        coordinates = np.load("./coord.npy")
+                    except:
+                        coordinates = None
+                    parallax.crop(original_img_path, coordinates, img_cropped_path)
+                    print('new photo to send')
+
+                    await manager.send_personal_message("true", websocket)
+
+            else:
+                await manager.send_personal_message("false", websocket)
+
+    except WebSocketDisconnect:
+        np.save('./coord.npy', None)
+        manager.disconnect(websocket)
+
+
+@app.post("/coord")
+async def post_coord(coordinates: Coordinates):
+    roomName = coordinates.roomName
+    img_cropped_path = './' + roomName + '/' + roomName + 'cropped.jpg'
+    original_img_path = './' + roomName + '/' + roomName + '.jpg'
+    coords = [[int(float(k[0])), int(float(k[1]))] for k in coordinates.coord]
+    np.save('./coord.npy', coords)
+
+    try:
+        parallax.crop(original_img_path, coords, img_cropped_path)
+
+    except Exception as e:
+        print(e)
```

Comment:
this will be lost in production. Catch a specific error or let it fail with a 500.